example/foo_bisector.py

```
@@ -6,7 +6,7 @@
 
 
 class FooBisector(GitBisector):
-    def get_output(self) -> str:
+    def get_output(self, cache_dir: Optiona[str] = None) -> str:
```

Ruff (0.8.2)

Undefined name `Optiona`

(F821)